### DISCUSSION TITLE
Is this plugin capable to do RAG from pdfs?

### DISCUSSION THREAD

**Discussion by gsuarezr** [pfrankov/obsidian-local-gpt#36] [Q&A]
I would love to run Retrieval-Augmented Generation on pdfs and be able to chat with it in obsidian, more specifically I would like the RAG sources to come from Zotero

**Comment by pfrankov** [comment]
Kind of
<img width="662" alt="image" src="https://github.com/user-attachments/assets/5b0a3121-f375-49ef-ad52-3b70585968cd">
